test/runtests.jl

```diff
@@ -596,6 +618,7 @@ model_types = (NonhydrostaticModel, HydrostaticFreeSurfaceModel)
                 end
 
             end
+            test_PEbuoyancytracer_PElineareos(grid)
```

Comment:
Same change needs to be made here

```suggestion
            test_PEbuoyancytracer_equals_PElineareos(grid)
```